root/about/meta_hack.html

```diff
@@ -0,0 +1,213 @@
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-sm-2 col-xs-12">
+        <a href="https://fastmail.com">
+          <img src="/static/images/sponsors/fastmail.png" style="background-color: #44557e; border-radius: 4px; padding-left: 6px;" class="img-responsive">
+        </a>
+      </div>
+      <div class="col-sm-10 col-xs-12">
+        <p>
+          <span><a href="https://www.fastmail.com/">FastMail</a></span><span>&nbsp;is a commercial hosted email service founded in 1999, which has established a reputation for technical leadership in the hosted email space, with a focus on security, privacy, and reliability. It is run by FastMail Pty Ltd, an Australian company based in Melbourne. From their early days they&#39;ve been users and supporters of Perl, and several of their developers are CPAN authors:</span><span><a href="https://metacpan.org/author/BRONG">&nbsp;</a></span><span><a href="https://metacpan.org/author/BRONG">BRONG</a></span><span>,</span><span><a href="https://metacpan.org/author/ROBN">&nbsp;</a></span><span><a href="https://metacpan.org/author/ROBN">ROBN</a></span><span>, and</span><span><a href="https://metacpan.org/author/ROBM">&nbsp;</a></span><span><a href="https://metacpan.org/author/ROBM">ROBM</a></span><span>&nbsp;(one of the founders). In late 2015 they</span><span><a href="https://blog.fastmail.com/2015/11/03/fastmail-acquires-pobox-and-listbox/">&nbsp;</a></span><span><a href="https://blog.fastmail.com/2015/11/03/fastmail-acquires-pobox-and-listbox/">acquired</a></span><span><a href="https://pobox.com/">&nbsp;</a></span><span><a href="https://pobox.com/">pobox.com</a></span><span>, another hosted mail company and longtime user and supporter of Perl. Pobox&#39;s tech team includes </span><span><a href="https://metacpan.org/author/RJBS">RJBS</a></span><span>&nbsp;and </span><span><a href="https://metacpan.org/author/WOLFSAGE">WOLFSAGE</a></span><span>.</span>
+        </p>
+      </div>
+    </div>
+
+
+    <div class="row">
+      <div class="col-md-12">
+        <h2 class="text-center">Silver Sponsors</h2>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-sm-2 col-xs-12">
+        <a href="https://opensource.careers/perl-careers/">
```

Comment:
While it (currently) redirects to the same place, please link to http://perl.careers/ instead everywhere you currently have this URL